Toast Stack: add demo description (#1834)
DevExpress/devextreme-demos: JSDemos/Demos/Toast/Stack/description.md

The DevExtreme Toast components support stacking. Use the [notify(message, stack)]() or [notify(options, stack)]() method to enable this feature.

> Within these methods, **stack** is an object with the following structure: *{position, direction}*.

Reviewer: ```suggestion
These methods use a **stack** object that have the following structure: *{position, direction}*.
```

## Specify Position

> You can set the `position` field to a string (see *'alias'* tab in the demo) or an object (see *'coordinates'* tab in the demo). Note that if you use an object for `position` field, you can only specify one field in that object for vertical coordinate and one field for horizontal coordinate (for example, if you specify *'top'*, you do not need to specify *'bottom'*).

Reviewer: ```suggestion
You can set the `position` field to a string (see *'alias'* tab in the demo) or an object (see *'coordinates'* tab in the demo). Note that the `position` field is an object, you can only specify one field in that object for vertical coordinate and one field for horizontal coordinate (for example, if you specify *'top'*, you do not need to specify *'bottom'*).
```

The DevExtreme Toast components can stack multiple notifications. Use the [notify(message, stack)](/Documentation/ApiReference/Common/Utils/ui/#notifymessage_stack) or [notify(options, stack)](/Documentation/ApiReference/Common/Utils/ui/#notifyoptions_stack) method to display stacked messages.

These methods use a **stack** object that has the following structure: *{position, direction}*.

## Specify Position

You can set the `position` field to a string (select *'predefined'* in the radio group) or an object (select *'coordinates'* in the radio group). Note that if you use coordinates for the `position` field, you need to specify one vertical and one horizontal coordinate only. For example, if you specify *'top'*, the demo disables the *'bottom'* field, and vice versa.

## Specify Direction

> The `direction` field specifies two options: which way the notification stack grows and whether new notifications appear at the end or in the beginning of the line. That's why you'll see pairs of values such as *'up-push'* and *'up-stack'*.

Reviewer 2: 1. ...`and whether new notifications appear at the end or in the beginning of the line.`

Please change "in the beginning" to "at the beginning".

2. `That's why you'll see pairs of values such as *'up-push'* and *'up-stack'*.`

I'd like to make this sentence sound less casual, and specify _what_ shows the *'up-push'* and *'up-stack'* values.
What do you think about the following?

`For this reason, the field's pull-down menu choices show pairs of values such as *'up-push'* and *'up-stack'*.`

> The DevExtreme Toast components support stacking. Use the [notify(message, stack)](/Documentation/ApiReference/Common/Utils/ui/#notifymessage_stack) or [notify(options, stack)](/Documentation/ApiReference/Common/Utils/ui/#notifyoptions_stack) method to enable this feature.

Reviewer 3: One flag for me is the ING in the first sentence: "support stacking". The second sentence ends with "enable this feature". 

In general, I try not to use "named features" in descriptions. We get into a habit of naming features, we know them all, and it's easy for us to refer to them like that. That's the way we think. I don't think our customers necessarily think the same way. Sometimes we assume that they know features by their names too. They jump in and out of our documentation with specific tasks - they are not interested (I think) in learning/remembering our feature names. Plus, usually sentences become a bit more complicated or vague when you use expressions like "the ABC feature". 

So here's my suggestion - just trying to be more specific and avoid INGs:

```suggestion
The DevExtreme Toast components can stack multiple notifications. Use the [notify(message, stack)](/Documentation/ApiReference/Common/Utils/ui/#notifymessage_stack) or [notify(options, stack)](/Documentation/ApiReference/Common/Utils/ui/#notifyoptions_stack) method to display stacked messages.
```

These methods use a **stack** object that has the following structure: *{position, direction}*.

## Specify Position

> You can set the `position` field to a string (see *'alias'* tab in the demo) or an object (see *'coordinates'* tab in the demo). Note that if you use coordinates for the `position` field, you need to specify one vertical and one horizontal coordinate only. For example, if you specify *'top'*, you do not need to specify *' bottom'*.

Author: We want to change the 'alias' and 'coordinates' titles of the radio group buttons.
Here are the suggestions:
'fixed' - 'absolute'
'fixed point' - 'coordinates'

Reviewer 3: I like 'coordinates'. For the other one, I would say 'alignment', because it lists possible alignment styles.

Author: I don't like 'alignment'. We have a title 'Position', and then there are tabs with 'alias' and 'coordinates'. If I change it to 'alignment' and 'coordinates', the word 'Position' doesn't fit. Besides, I think that 'alignment' is used usually with text or in the context of aligning blocks in CSS horizontally or vertically. In the demo, we don't align toasts, we position them.

What do you think about the following?
Position
- relative    - coordinates

Reviewer 3: My issue with "relative" is that it's not necessarily opposite to "coordinates". Coordinates may be relative. 

One other thing I can suggest is "Position: predefined | coordinates". That would imply we offer a few "predefined" choices like "top-right" or "bottom-left" and we allow to specify exact coordinates. 

And if you don't like that, then I guess it's okay to go with "relative" and "coordinates". It's not a big deal.

> You can set the `position` field to a string (see *'alias'* tab in the demo) or an object (see *'coordinates'* tab in the demo). Note that if you use coordinates for the `position` field, you need to specify one vertical and one horizontal coordinate only. For example, if you specify *'top'*, you do not need to specify *' bottom'*.

Reviewer 3: ```suggestion
You can set the `position` field to a string (select *'alignment'* in the radio group) or an object (select *'coordinates'* in the radio group). Note that if you use coordinates for the `position` field, you need to specify one vertical and one horizontal coordinate only. For example, if you specify *'top'*, the demo disables the *'bottom'* field, and vice versa.
```

## Specify Direction

> The `direction` field values can be regular and reverse. For example, *'up'* and *'up-reverse'*.

Reviewer 3: ```suggestion
The `direction` field specifies two options: which way the notification stack grows and whether new notifications appear at the end or in the beginning of the line. That's why you'll see pairs of values such as *'up'* and *'up-reverse'*.
```

- *'up'*   

> Subsequent toasts push the previous toast upwards.

Reviewer 3: ```suggestion
New toasts push the previous toasts upwards.
```

- *'up-reverse'*    
Toasts stack on top of each other. 

> This behavior applies to other directions.

Reviewer 3: I'd say you don't need this line. I think the introduction before the list covers it.
